examples/custom_plot_manipulation/src/main.rs

```diff
@@ -22,6 +24,8 @@ struct PlotExample {
     shift_to_horizontal: bool,
     zoom_speed: f32,
     scroll_speed: f32,
+    gradient_color: bool,
+    gradient_fill: bool,
```

Comment:
From the description of this example:

> This example shows how to implement custom gestures to pan and zoom in the plot

In other words - I don't think gradient stuff belong here at all

Reply:
Good point. Reverted changes in the latest commit